src/NHibernate.DomainModel/CustomPersister.cs

```diff
@@ -49,7 +49,7 @@ public string RootEntityName
 
 		public string EntityName
 		{
-			get { return typeof (Custom).FullName; }
+			get { return typeof(Custom).FullName; }
```

Comment:
Please revert this and others formatting only changes, in order to maintain a cleaner git history.

Reply:
ok